src/main/java/com/github/kay9/dragonmounts/dragon/TameableDragon.java

```diff
@@ -146,8 +152,8 @@ protected void registerGoals() // TODO: Much Smarter AI and features
 //        goalSelector.addGoal(1, new DragonLandGoal(this));
         goalSelector.addGoal(2, new SitWhenOrderedToGoal(this));
         goalSelector.addGoal(3, new MeleeAttackGoal(this, 1, true));
-//        goalSelector.addGoal(4, new DragonBabuFollowParent(this, 10));
-        goalSelector.addGoal(5, new FollowOwnerGoal(this, 1.1, 10f, 3.5f, true));
+        goalSelector.addGoal(4, new DragonFollowParentGoal(this, 1d));
```

Comment:
I was just matching the commented out goal originally, but I think if the follow owner goal comes first, babies will never follow parents unless we make some way to toggle between them.

Mostly thinking that because the only thing that stops the follow owner is to make them sit which would stop them from following the parent.

I'd say if you want the owner follow prioritized, I'll just remove the parent follow goal for now and look at making a custom state change between follow owner, sit, and follow parent.

Reply:
I guess I figured the baby would follow the parent when the player and parent are both within owner following distance. Thinking about it now it does feel like a niche use case and would probably happen less often than intended.

I think the best approach to this issue would be to follow vanilla's way of going about it, **and to scrap the feature all together, unfortunately.** It seems vanilla does not incorporate parent following to tameable animals (other than horses, which don't follow owners anyway.)

Reply:
Yeah, I think that's fair. Dragons don't need to love their parents anyways. They have their owners!